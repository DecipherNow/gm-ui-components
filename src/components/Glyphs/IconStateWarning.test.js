import React from "react";
import { shallow } from "enzyme";
import IconStateWarning from "./IconStateWarning.svg";
describe("IconStateWarning", () => {
  it("matches snapshot", () => {
    
    const aIconStateWarning = shallow(<IconStateWarning />).find("IconStateWarning.svg").dive();
    console.log(aIconStateWarning.debug())
    expect(aIconStateWarning).toMatchSnapshot();
  });
});