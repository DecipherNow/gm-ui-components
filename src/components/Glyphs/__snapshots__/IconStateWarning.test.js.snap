// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconStateWarning matches snapshot 1`] = `<IconMock />`;
